server/express/package.json

```diff
@@ -13,10 +14,14 @@
     "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
+    "fs": "^0.0.1-security",
     "morgan": "^1.10.0",
     "passport": "^0.4.1",
-    "passport-did-auth": "^1.0.0-alpha.7",
+    "passport-did-auth": "1.0.0-alpha.18",
```

Comment:
I think it would be good to upgrade to latest passport-did-auth here (1.0.0-alpha.23)